doc/qbittorrent.1.md

```diff
@@ -1,36 +1,236 @@
+
+qBittorrent can be cleanly terminated by fully closing the GUI (including the system tray, if that option is enabled) or by sending the **SIGINT** or **SIGTERM** signals.
 
 # OPTIONS
-**`--help`** Prints the command line options.
 
-**`--version`** Prints qbittorrent program version number.
+## Generic program information
+
+`-h`, `--help`
+
+:    Display a short help text about the command-line options and exit.
+
+`-v`, `--version`
+
+:    Display program version and exit.
+
+## General configuration
+
+`--configuration=`_NAME_
+
+:    Use a different configuration with name _NAME_. This enables running multiple separate instances of qBittorrent at the same time without conflict (provided they don't listen on the same ports for WebUI access, incoming connections, etc). Configuration and data files will be stored under a configuration hierarchy similar to the default one (see the _FILES_ section for more information) but with a final directory named **qBittorrent\__NAME_** instead of **qBittorrent**. For example, while the main configuration file is stored as _~/.config/qBittorrent/qBittorrent.conf_, the configuration file for a configuration named **foo** will be stored as _~/.config/qBittorrent\_**foo**/qBittorrent.conf_. If a configuration with the given _NAME_ does not exist yet, it is created with default settings. If it already exists, it is loaded as a normal configuration would be.
+
+`--no-splash`
+
+:    Do not display the splash screen on startup.
+
+`--profile=`_PATH_
+
+:    Store configuration _FILES_ in _PATH_. This enables what is known as "portable mode". Similarly to **`--configuration`**, this option can be used to run multiple separate instances of qBittorrent at the same time without conflitcts (the same caveats about listening port conflicts and other potential unavoidable conflicts apply). The difference is that configuration files are all stored together under the specified configuration _PATH_, instead of scattered under a separate directory hierarchy in the system's configuration directories (see the _FILES_ section for more information). This enables one to migrate their entire configuration from system to system by simply copying around one directory. If _PATH_ does not exist, it is created, with configuration files initialized with default settings inside it that a new instance of qBittorrent loads. If it exists, the configurations are loaded from there. A relative _PATH_ may be used, in which case it will be relative to the current working directory that qBittorrent is started from. Note that **`~`** will not get expanded, and will be interpreted as a folder literally named **`~`**. If both **`--configuration`** and **`--profile`** are specified, only **`--profile`** takes effect. Implies **`--relative-fastresume`**.
+
+`--relative-fastresume`
+
+:    Hack into libtorrent _.fastresume_ files and make file paths relative to the **`--profile`** directory. "If you have to ask, you don't need this."
```

Comment:
>maybe rephrase it?

You really need to rephrase it in a end user friendly manner (without digging into internals).

Reply:
@Chocobo1
@glassez 
Isn't the `"If you have to ask, you don't need this."` disclaimer at the end sufficient?
I think the usage of "hack" here is appropriate in this case. This is the same description that has been used in the `-h` output so far.

Also the official `git` documentation, for example, uses it extensively, so this is not really breaking precedent.

Reply:
IMO, even name of this option is bad.
Really it stores file paths in relative form.
The fact that qBittorrent uses .fastresume files to save some torrent settings is a temporary phenomenon even though it has been around for a very long time. We can change this at any time, but this will not affect the meaning of this option.

Reply:
@glassez so should I change the name of this option in this PR? IMO it should be done in another one.

Reply:
>should I change the name of this option in this PR? 

I think it should be done in separated PR.
But the following should be here:
>You really need to rephrase it in an end user friendly manner (without digging into internals).

Reply:
Fixed.